### DISCUSSION TITLE
Post Process Effects not working on IOs

### DISCUSSION THREAD

**Discussion by gabrielgaldinomagalhaes** [google/model-viewer#4292] [Q&A]
I'm working with Post Process Effects to create some bulb light effects. I was able to make it work fine on Android. But when talking about iOS (tested on iPhone and on iPad), it doesn't seem to work fine. I saw in the documentation that:

> Safari does not support import maps

But it also says:

> Safari does not support import maps, the following script will make your code work universally:

![image](https://github.com/google/model-viewer/assets/45290323/2bd8bb17-de61-453a-8427-73e22ce51f5a)

I added all the codes, the effects work well in the Android, but it still doesn't work in iOS. What might be happening ?

**Comment by elalish** [comment]
So many possibilities - have you checked the JS console for errors or warnings? Do you have a live link?

**Comment by BenSztukowski** [comment]
I haven't been able to get post processing working through GWD, using the same steps it seems like requests for those :

 Failed to load resource: the server responded with a status of 404 (Not Found) model-viewer-module.min.js:1
 Failed to load resource: the server responded with a status of 404 (Not Found) model-viewer-effects.min.js:1
 
 When pasting the URLs into a browser, it gives a 404 as well
If inspecting the examples page, it looks like the min.js files are locally stored on the server. Maybe could try localizing those two files?

**Reply by elalish** [reply]
You mean Google Web Designer? This feature is still pre-release in MV. Also, last I checked GWD is woefully out-of-date on its MV version, so this may take some time. Please let them know you want the latest MV!

**Reply by BenSztukowski** [reply]
Just gave em' a question about it; I'll try and be patient for the features!
